_layouts/help.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: main
+---
+{% capture slug_permalink %}/{{ page.slug }}{% endcapture %}
+{% assign parent_url = page.url | replace: slug_permalink %}
+{% assign related_questions = site.help | where_exp: "item", "item.url contains parent_url" | sort: 'order' %}
+{% assign this_page_parent = '' %}
+
+{% include help/nav_page_top_small.html %}
+{% include help/nav_page_top.html %}
+
+<div class="bg-white">
+  <div class="container cntnr-wide px2 pt2 pb0">
+    <div class="clearfix">
+      {% include help/nav_sidenav.html %}
+      <div class="sm-col sm-col-8 sm-pl2 sm-pr5">
+        <h2 id="{{ page.slug }}" class="mt0 mb1 pt2">
+          {{ page.title }}
+        </h2>
+        <img alt="hr" src="{{ site.baseurl }}/assets/img/hr-red-{{ _index }}.svg" height="6">
+        <div class="mb0">
+          <h2 id="{{ question.title | slugify }}" class="mt0 mb1 pt2">
```

Comment:
This `h2` can be removed since we aren't using the question data file any longer - this is just results in a blank `h2` at the moment.